src/main/java/Duke/DeleteCommand.java

```diff
@@ -0,0 +1,51 @@
+package Duke;
+
+import java.io.IOException;
+
+/**
+ * The DeleteCommand class represents a command to delete a task from the TaskList.
+ * It implements the Command interface.
+ */
+public class DeleteCommand implements Command {
```

Comment:
I think it would be better to name this CommandDeleter (according to the naming standards)